lxc/file.go

```diff
@@ -69,6 +70,145 @@ func (c *fileCmd) flags() {
 	gnuflag.BoolVar(&c.mkdirs, "p", false, i18n.G("Create any directories necessary"))
 }
 
+func (c *fileCmd) recursivePullFile(d lxd.ContainerServer, container string, p string, targetDir string) error {
+	buf, resp, err := d.GetContainerFile(container, p)
+	if err != nil {
+		return err
+	}
+
+	target := filepath.Join(targetDir, filepath.Base(p))
+	if resp.Type == "directory" {
+		if err := os.Mkdir(target, os.FileMode(resp.Mode)); err != nil {
```

Comment:
It's code I moved from client.go and my OCD didn't seem to trigger on that one, fixing :)